src/lib.rs

```diff
@@ -348,6 +352,92 @@ pub(crate) trait Castable {
     type RustType;
 }
 
+/// Defines a new [`Castable`] opaque struct with [`OwnershipBox`] ownership.
+///
+/// Expects to be invoked with a visibility specifier, the struct keyword, a struct name, and
+/// in parens, the Rust type that the opaque struct wraps. Similar to defining a normal struct
+/// that wraps a single type as its `.0` member.
```

Comment:
I would say "the Rust type pointed to by raw pointers of the opaque struct's type."

For the second sentence, I would say something like:

> Similar to a [newtype](https://doc.rust-lang.org/book/ch19-04-advanced-types.html#using-the-newtype-pattern-for-type-safety-and-abstraction) with `#[repr(transparent)]`, but only allows conversions between Box and raw pointers.

(and similar in other comments with this verbiage)

Reply:
Fixed up in each case.